lib/datatrans/json/transaction/merchant_authorize.rb

```diff
@@ -0,0 +1,45 @@
+require 'httparty'
+require 'datatrans/json/transaction/response'
+
+class Datatrans::JSON::Transaction
+  class MerchantAuthorize
```

Comment:
> Datatrans::JSON::Transaction::Authorize is actually not for the [authorize](https://api-reference.datatrans.ch/#tag/v1transactions/operation/authorize) operation but rather for the [init](https://api-reference.datatrans.ch/#tag/v1transactions/operation/init) operation.

yeah this rather confusing ... I propose to rename `Datatrans::JSON::Transaction::Authorize` to `Datatrans::JSON::Transaction::Initialize`, but create an alias for it (for now).

Once we release the breaking change where the XML API is removed, we can alias this class to `Authorize`.

Reply:
I think I know what you mean but I'm not sure in what order it would be best to rename / alias / deprecate... If you propose something, I can review it 🙏